content/en/tracing/llm_observability/span_kinds.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Span kinds
+kind: documentation
+---
+
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+LLM Observability supports the following span kinds:
+
+| Kind      | Represents   | Root span?   | Can have child spans? |
+|-----------|--------------|--------------|-------------|
+| [Agent](#agent-span)     | A series of choices made by a language model, often consisting of tool calls and calls to external knowledge bases | Yes | Yes  |
+| [Workflow](#workflow-span)  | A predetermined sequence of steps, often consisting of some combination of tool calls and data retrieval | Yes | Yes |
+| [LLM](#llm-span)      | Individual LLM calls, such as a call to OpenAI GPT-4 | Yes | No |
+| [Tool](#tool-span)      | Function calls to external programs or services, such as a web search API | No | No |
+| [Embedding](#embedding-span) | A call to an embedding model to create an embedding suitable for sending into a subsequent retrieval step | No  | Yes |
+| [Retrieval](#retrieval-span) | An array of documents returned from an external knowledge base | No | No | 
+| [Task](#task-span)      | A step in the chain that does not involve a call to an external service, such as a data preprocessing step | No | No |
+
+For instructions on creating spans from your application, including code examples, see [Tracing spans][2] in the SDK documentation.
+
+## Agent span
+
+Agent spans represent a dynamic sequence of operations where an autonomous decision-maker determines which operations and tools to execute. For example, an agent span might represent a series of reasoning steps controlled by a [ReAct agent][1].
```

Comment:
"where an autonomous decision-maker determines which operations and tools to execute" -> "where a large language model determines which operations to execute"

Mainly because:
- tools is a subset of operations
- we don't want to confuse users with 'autonomous decision-maker', our intended users are just building LLM apps